packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -114,8 +112,13 @@ $close-button-width: tokens.bpk-spacing-lg() * 2;
     position: sticky;
     top: 0;
     z-index: tokens.$bpk-zindex-tooltip - 1;
+    padding: tokens.bpk-spacing-sm() 0;
```

Comment:
Adding additional padding to the heading based on BottomSheet design (avoiding adding to Nav Bar directly to avoid all other consumers gaining height)